### PR TITLE
Milestone 4

### PR DESCRIPTION
**Pull request by wandji20**
in this Milestone, we have:
- created and implemented a filter reducer and
- created a filter function to filter books based on the filter from the filter reducer.

### CODE REVIEW

File: src/reducers/filterReducer.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { CHANGE_FILTER } from '../actions';
+
+const initialState = {
+  filter: '',
+};
+
+const changeFilterReducer = (state = initialState, action) => {
```
Comment (bellom) [inline src/reducers/filterReducer.js:7]: Oops, we need to set the state to `All`, that is; users are meant to see all books at first load.
image

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi Team :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best**

**Review by mahmoud717 — APPROVED** [review]
### STATUS: APPROVED ✔️ ✔️ 

**hi @wandji20  & @SamArsanios**

**Great job with making all of the requirements. the project is approved, you can go ahead and merge this pull request**

File: src/reducers/filterReducer.js (hunk 1 above)
Comment (mahmoud717) [reply src/reducers/filterReducer.js:7]: Good job guys

**Pull request merged**